clearcut_detection_backend/frontend/src/components/Switch/index.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+
+import './style.css';
+
+const Switch = ({ state, handleClick }) => {
+  const renderIcon = () => (
+    <div className="switch-icon">
+      <div></div>
+      <div></div>
+      <div></div>
+    </div>
+  );
+
+  return (
```

Comment:
plase, make align cetner for sandwich and logo

![image](https://user-images.githubusercontent.com/23707448/68470454-c8d98c00-0224-11ea-95b9-c766b9999436.png)